pkg/apis/serving/v1/revision_validation_test.go

```diff
@@ -383,6 +383,20 @@ func TestRevisionSpecValidation(t *testing.T) {
 				}},
 			},
 		},
+		wc: func(ctx context.Context) context.Context {
+			s := config.NewStore(logtesting.TestLogger(t))
+			s.OnConfigChanged(&corev1.ConfigMap{ObjectMeta: metav1.ObjectMeta{Name: autoscalerconfig.ConfigName}})
+			s.OnConfigChanged(&corev1.ConfigMap{ObjectMeta: metav1.ObjectMeta{Name: config.DefaultsConfigName}})
+			s.OnConfigChanged(&corev1.ConfigMap{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: config.FeaturesConfigName,
+				},
+				Data: map[string]string{
+					"multi-container": "disabled",
+				},
+			})
+			return s.ToContext(ctx)
+		},
 		want: &apis.FieldError{Message: "multi-container is off, but found 2 containers"},
```

Comment:
I see this testcase is for disabled flag.
Can we also add a testcase for positive scenario?
I mean now by default flag is enabled so if we give multiple containers then we should get a success response.

Reply:
You're right, I will add a testcase for positive scenario
Do I need to add a testcase that does not specify a `ContainerPort` with a fail response?

Reply:
Sorry, I didn't see it, there are already related test cases